identity/app/views/fragments/newsletterSwitch.scala.html

```diff
@@ -56,5 +56,5 @@
     extraFields = Nil,
     footer = Some(buildFooter(newsletter)),
     skin = skin,
-    newsletterIdentityName = Some(newsletter.identityName)
+    newsletterIdentityName = Some(newsletter.id)
```

Comment:
this is equivalent
https://github.com/guardian/identity/blob/main/identity-model/src/main/scala/com/gu/identity/model/Serialisation.scala#L235